workflow_scheduler.go

```diff
@@ -56,7 +66,7 @@ func newWorkflowScheduler(schedulesJson []byte, workflowFactory WorkflowFactory)
 				panic("Given schedule type not supported.")
```

Comment:
This shouldn't be a panic really. We should just log and continue with the remaining schedules. Something to fix in a future PR.

Reply:
I see, fixed in current PR.